### ISSUE TITLE
Close tab button nested in tab element causing misinterpretation (4.1.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux-tabs#88]
### Issue from RE NXT A11y audit 2019
**Ensure markup documents contain well-formed elements**

The Close Tab button is enclosed with an ARIA page tab element. This is not valid.

### Impact
The page tab name is not announced properly causing confusion. For example, the Overview tab is announced as "Overview Close Tab Tab"

[Code]
<a _ngcontent-c6="" class="sky-btn-tab sky-tab-btn-closeable" role="tab" tabindex="0" aria-selected="false">
  Overview
  
  <button _ngcontent-c6="" class="sky-btn-tab-close" type="button" aria-label="Close tab">
    <sky-icon _ngcontent-c6="" icon="remove" _nghost-c7=""><i _ngcontent-c7="" aria-hidden="true" class="fa sky-icon fa-remove"></i>
</sky-icon>
  </button>
</a>

### Recommedation
The Close Tab button should be outside of the tab element and have a meaningful label as "Close Overview Tab"

[AB#1564727](https://blackbaud.visualstudio.com/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/1564727)
